### DISCUSSION TITLE
winget list stopped working

### DISCUSSION THREAD

**Discussion by awg43** [microsoft/winget-cli#2096] [Q&A]
winget search works fine. However winget starts crolling to 100% and then stops.
I found next report (dutch translated to english)

Faulting application name: winget.exe, version: 1.17.2201.27001, timestamp: 0x61f32397
Faulting module name: WindowsPackageManager.dll, version: 1.17.2201.27001, timestamp: 0x61f32322
Exception code: 0xc0000005
Fault offset: 0x0000000000089bf3
Faulting process id: 0x2e98
Faulting application start time: 0x01d85039585b9459
Faulting application path: C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.17.10271.0_x64__8wekyb3d8bbwe\winget.exe
Faulting module path: C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.17.10271.0_x64__8wekyb3d8bbwe\WindowsPackageManager.dll
Report-id: 57497ee6-7212-4f6e-acb4-2f9eef05b38b
Faulting package full name: Microsoft.DesktopAppInstaller_1.17.10271.0_x64__8wekyb3d8bbwe
Faulting package-relative application ID: winget

How to resolve this?
Thanks in advance!

**Comment by KnifMelti** [comment]
I don't know, but I have the exact same issue here, but only in System context!
As an admin user everything is fine...

Win 10: 21H2 (19044.1645)
C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.17.10271.0_x64__8wekyb3d8bbwe\Winget.exe (v1.2.10271)

When running in an admin user context there's no problem at all...
...when running in system context (https://www.nirsoft.net/utils/advanced_run.html) the output is empty (due to crash)!
"winget.exe source reset --force" doesn't help

**Reply by KnifMelti** [reply]
...and, the same goes for "winget export"!

**Reply by KnifMelti** [reply]
More details:
https://github.com/Romanitho/Winget-Install/issues/22

**Reply by EricPez** [reply]
hello I am having exactly the same issue. Do someone has already found a way to fix that ? or shall we have to wait on Microsoft to release a fix with an upcoming Windows Update ?

**Reply by KnifMelti** [reply]
> 

Winget latest preview release solution to the crash in winget list/export

**Comment by KnifMelti** [comment]
@awg43 Winget latest preview release solution to the crash in winget list/export
